app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user
+
+  def update_post_counter
+    user.increment!(:post_counter)
+  end
```

Comment:
- [ ] OPTIONAL: You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private:

```ruby
# app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    user.increment!(:posts_counter)
  end
```

Same refers to other counters 😉

Reply:
Thank you very much @ShoiraTa 🤗 . I'd update